src/components/ProjectMemberCard/ProjectMemberCard.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  Card,
+  CardActions,
+  CardContent,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import { FC } from 'react';
+import * as React from 'react';
+import { displayRole } from '../../api/displayRole';
+import { displayUserName } from '../../api/displayUserName';
+import { Avatar } from '../Avatar';
+import { useDateFormatter } from '../Formatters';
+import { ButtonLink } from '../Routing';
+import { ProjectMemberFragment } from './ProjectMember.generated';
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
+  cardContent: {
+    display: 'flex',
+  },
+  cardActions: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    borderTop: `0.5px solid ${palette.divider}`,
+    padding: spacing(1, 2, 1, 1),
+  },
+  avatar: {
+    width: spacing(7),
+    height: spacing(7),
+    marginRight: spacing(2),
+  },
+  memberInfo: {
+    flexGrow: 1,
+  },
+}));
+
+export interface ProjectMemberCardProps {
+  imageSource?: string;
+  projectMember?: ProjectMemberFragment;
+  // TODO this should use primary organization on User when api is finished
+  primaryOrganizationName?: string;
+}
+
+export const ProjectMemberCard: FC<ProjectMemberCardProps> = ({
+  projectMember,
+  imageSource,
+  primaryOrganizationName,
+}) => {
+  const classes = useStyles();
+  const dateFormatter = useDateFormatter();
+
+  const name = projectMember?.user.value
+    ? displayUserName(projectMember.user.value)
+    : '';
+  const rolesString = projectMember?.roles
+    ? displayRole(projectMember.roles)
+    : '';
+  const createdAtString = projectMember?.createdAt
```

Comment:
`createdAt` is a `DateTime` - use the `useDateTimeFormatter`.

Also if you want to update those to accept and return nullable that would probably be useful too.